### ISSUE TITLE
Getting depreciation warnings on launch

### ISSUE THREAD

**Issue by felixmr1** [LunarVim/LunarVim#4377]
### Problem description

On a fresh install, just launch lunarvim and I get

```
~/.local/share/lunarvim/lvim/lua/lvim/config/init.lua:72: `lvim.lsp.automatic_servers_installation`
 is deprecated.
~/.local/share/lunarvim/lvim/lua/lvim/config/_deprecated.lua:175: `as` in `lvim.plugins` is deprecated.
 Use `name` instead. See https://github.com/folke/lazy.nvim#-migration-guide.
```

### LunarVim version

release-1.3/neovim-0.9-d1c1bace-dirty

### Neovim version (>= 0.9.1)

NVIM v0.9.1

### Terminal name

Alacritty

### Operating system/version

PopOS 22.04

### Steps to reproduce

As description says

**Comment by cpea2506** [comment]
Do you have any of your config before? Then you need to change which are deprecated.

**Comment by cpea2506** [comment]
> Sorry for late answer. But I did not have anything before, this was a fresh install.

I think maybe you can do a fresh install again or follow the suggestion on the message to suppress it.

**Comment by felixmr1** [comment]
@cpea2506 I managed to figure it out.

What confused me was that it was throwing errors from functions deep in lunarvims installation folder. Like `~/.local/share/lunarvim/lvim/lua/lvim/config/init.lua:72` when the error was actually in my own lua config.

So I followed the folke migration guide for my user plugins: https://github.com/folke/lazy.nvim#-migration-guide

And updated `lvim.lsp.automatic_servers_installation` to `lvim.lsp.installer.setup.automatic_installation`.

Which fixed both of the warnings!
